pkg/trace/trace.go

```diff
@@ -38,6 +38,7 @@ type RequestInfo struct {
 	RawQuery string      `json:"rawquery,omitempty"`
 	Headers  http.Header `json:"headers,omitempty"`
 	Body     []byte      `json:"body,omitempty"`
+	Client   string      `json:"client"`
```

Comment:
It looks like the `Client` field is just added here and a value passed in the other file, but the value is never read. If it is not going to be read, we don't need to add it in the struct. Can it be removed?

Reply:
@donatello , this field is used by `mc admin trace`

Reply:
Ah it's in the output.